Minor correction to documentation

LeahPrice opened the pull request:
Correcting the description of GetParticleWeight().

eddelbuettel commented on ChangeLog:
```diff
@@ -1,4 +1,8 @@
+
+    * inst/include/sampler.h: Correcting the description of GetParticleWeight().
+    
+2017-07-24  Leah South  <leah.south@hdr.qut.edu.au>
```
Nitwick: when one has two entries _with the same data and author_ one tends to merge those.

But thanks for keeping up with ChangeLog!

LeahPrice replied:
Sorry are you referring to merging the entries in the ChangeLog or another type of merge? I just did a commit to remove the unnecessary ChangeLog entry.

eddelbuettel commented:
@LeahPrice You removed the entire ChangeLog entry.  That is not what I suggested. I suggested to regroup all entries having the same date under the same date.  Anyway....

LeahPrice commented:
@eddelbuettel Okay, thanks for clarifying that. When you said the same data and author, I misinterpreted it as the same author and general theme of the update. I removed the extra entry because it was just an error introduced in the previous change.

I've fixed this up now and combined into one.

eddelbuettel commented:
Merged.  I still think you need entries for your name and email in `~/.gitconfig`:

```sh
edd@brad:~$ head -3 .gitconfig
[user]
        name = Dirk Eddelbuettel
        email = edd@debian.org
edd@brad:~$
```

so that your commits are market as yours.  Just about every git beginner's tutorial mentions this, you may have skipped it.  Might be worth adding (though again I have no idea where to do this on Windows, but lots of other users seem to manage....)

LeahPrice commented:
Thanks @eddelbuettel for picking up that it didn't work. Could you please let me know where you found that quoted log that said <unknown> so I can follow it up? The Gitkraken tutorial says it happens automatically - updating my profile didn't work but surely the manual fix will.

eddelbuettel commented:
`git log` on the command shell.  I use an alias that pretty-prints the log output (and there are many).
